libs/cdk/src/lib/utils/directives/focusable-list/focusable-list.directive.ts

```diff
@@ -26,22 +60,54 @@ export class DeprecatedFocusableListDirective extends DeprecatedSelector {}
     selector: '[fdkFocusableList]',
     exportAs: 'fdkFocusableList',
     standalone: true,
-    providers: [FocusableListService, DestroyedService]
+    providers: [
+        {
+            provide: FDK_FOCUSABLE_LIST_DIRECTIVE,
+            useExisting: FocusableListDirective
+        },
+        DestroyedService
+    ]
 })
 export class FocusableListDirective implements AfterViewInit {
-    /** Direction of navigation. */
+    /** Direction of navigation. Always horizontal when in grid. */
```

Comment:
In my opinion a third option here like `'grid'` would make more sense

Reply:
Good idea!